Photon-Tinker/DeviceInspectorDataViewController.swift

```diff
@@ -56,7 +55,7 @@ class DeviceInspectorDataViewController: DeviceInspectorChildViewController, UIT
     
     
     override func viewWillDisappear(_ animated: Bool) {
-        IQKeyboardManager.shared().shouldHidePreviousNext = false
+        IQKeyboardManager.shared().previousNextDisplayMode = .default
```

Comment:
Think we should continue using IQKeyboardManager? thought you didn't like it

Reply:
I didn't like the way keyboard behaved. We thought the reason for that was IQKeyboardManager, it turns out it was just a problem with configuration and some usage limitations which I believe I was able to overcome.